traefik.go

```diff
@@ -79,17 +86,25 @@ func main() {
 
 	// listen new configurations from providers
 	go func() {
+
 		for {
-			configuration := <-configurationChan
-			log.Infof("Configuration receveived %+v", configuration)
-			if configuration == nil {
+			configMsg := <-configurationChan
+			log.Infof("Configuration receveived from provider %v: %+v", configMsg.providerName, configMsg.configuration)
+			if configMsg.configuration == nil {
 				log.Info("Skipping empty configuration")
-			} else if reflect.DeepEqual(currentConfiguration, configuration) {
```

Comment:
I think you may be looking at an old commit, currentConfiguration is not present in the latest push.  New line is: 
```go
    } else if reflect.DeepEqual(currentConfigurations[configMsg.providerName], configMsg.configuration) {
```